pkg/sql/schema_changer.go

```diff
@@ -1026,7 +1026,7 @@ func (sc *SchemaChanger) createRollbackJob(
 			}
 			payload := job.Payload()
 			rollbackJob := sc.jobRegistry.NewJob(jobs.Record{
-				Description:   "ROLL BACK " + payload.Description,
+				Description:   fmt.Sprintf("ROLL BACK JOB: %d, ", *job.ID()) + payload.Description,
```

Comment:
This makes the output something like `ROLL BACK JOB: 123 ALTER TABLE BLAH`. Maybe `ROLL BACK JOB 123: ALTER TABLE BLAH` is more readable?

Reply:
But I don't care too much either way.